#this attaches a bounding box of the given object detection to the indicated end effector

string arm_name
bool remove

mapping_msgs/CollisionObject object
---
---
